.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
   # so we manually upgrade to a known working npm version.
   - npm install --global npm@4
   - 'if [ "$JOB" = "test" ]; then make test-ci; fi'
-  - 'if [ "$JOB" = "lint" ]; then make lint && make flow; fi'
+  - 'if [ "$JOB" = "lint" ]; then make build && make lint && make flow; fi'
```

Comment:
bleh. probably would be better to build _just_ the plugin needed for linting

Reply:
could just make it "internal" and not a real "package", and not build in es6 if we want to skip the build step

Reply:
example: https://github.com/babel/babel/blob/960151c8766c42e12c2e45d3f9c032aea665b954/.babelrc.js#L10 (and actually we can remove that plugin now anyways)

Reply:
think that seems like the way to go! pushed something along those lines up. let me know what you think, particularly in regards to the file structure. not sure how i feel about the `internal/` directory i added